.clang-format

```diff
@@ -67,21 +64,11 @@ SortIncludes : true
 SortUsingDeclarations: true
 SpaceAfterTemplateKeyword: false
 SpaceBeforeAssignmentOperators: true
-
-#Unsupported in VS2017-included ClangFormat 6.0 we'll see about VS2019's ClangFormat 7.0
-#SpaceBeforeCpp11BracedList: false
-
-#Unsupported in VS2017-included ClangFormat 6.0 we'll see about VS2019's ClangFormat 7.0
-#SpaceBeforeCtorInitializerColon: true
-
-#Unsupported in VS2017-included ClangFormat 6.0 we'll see about VS2019's ClangFormat 7.0
-#SpaceBeforeInheritanceColon: true
-
+SpaceBeforeCpp11BracedList: false
+SpaceBeforeCtorInitializerColon: true
+SpaceBeforeInheritanceColon: true
 SpaceBeforeParens: ControlStatements
-
-#Unsupported in VS2017-included ClangFormat 6.0 we'll see about VS2019's ClangFormat 7.0
-#SpaceBeforeRangeBasedForLoopColon: false
-
+SpaceBeforeRangeBasedForLoopColon: false
```

Comment:
I suspect we want this to be `true`. Existing uses of range for put a space before the colon.

Reply:
Point. Oversight on my part.